src/test/java/com/hivemq/cli/commands/swarm/run/SwarmRunStartCommandTest.java

```diff
@@ -190,12 +182,12 @@ void uploadScenario_vm_error(final @TempDir @NotNull Path tempDir) throws IOExce
     }
 
     @Test
-    void uploadScenarioSuccessButCantStart_error(final @TempDir @NotNull Path tempDir) throws IOException, ApiException {
+    void uploadScenarioSuccessButCantStart_error(final @TempDir @NotNull Path tempDir)
+            throws IOException, ApiException {
```

Comment:
```suggestion
            throws Exception {
```

Reply:
IMO this does not reduce readability but improves clearance. As it is a bad practice to bundle Exceptions in production code I did not change it in test code.